.github/workflows/ci.yml

```diff
@@ -71,20 +71,30 @@ jobs:
           (cd doc; opam exec -- make)
           (cd tests; opam exec -- make)
 
-  lint-opam:
+  lint-fmt:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout tree
         uses: actions/checkout@v4
-
       - name: Set-up OCaml
         uses: ocaml/setup-ocaml@v3
         with:
-          ocaml-compiler: "4.14"
-          dune-cache: true
```

Comment:
Since a dune cache will be created for each job, you may want to set it only for the main job to avoid cache thrashing due to GitHub's cache size limit.